template/.github/workflows/nightly_at_main.yml

```diff
@@ -32,3 +32,24 @@ jobs:
       python-version: ${{ matrix.python.version }}
       tox-env: ${{ matrix.python.tox-env }}
     secrets: inherit
+
+  tests-at-lower-bound:
+    name: Tests at lower bound
+    needs: setup
+    strategy:
+      matrix:
+        os: ['ubuntu-24.04']
+        python:
+          - version: '${{needs.setup.outputs.min_python}}'
+    runs-on: ${{ matrix.os }}
+    env:
+        ESS_PROTECTED_FILESTORE_USERNAME: ${{ secrets.ESS_PROTECTED_FILESTORE_USERNAME }}
+        ESS_PROTECTED_FILESTORE_PASSWORD: ${{ secrets.ESS_PROTECTED_FILESTORE_PASSWORD }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python.version }}
+      - run: python -m pip install --upgrade uv pytest
```

Comment:
[minor]
`[test]` environment should already have `pytest`... I think...? So maybe no need to install pytest here.